doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,51 @@
+# Use Case: Specification for application communicating with multiple services
+
+## Summary
+
+- **Scope:** Configuration
+- **Level:** Developer Goal
+- **Actors:** Application developer
+- **Brief:** The user wants to specify the port and url for an unknown number of servers, which the application shall use.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:**
+  - Specification is written and stored system-wide or user specific.
+  - All services which are added in the configuration need to adhere to this configuration specification.
+- **Alternative scenario:** None.
+- **Error scenario:**
+  - In case services have overlapping configuration an error is yielded.
```

Comment:
Do you mean specification here? Please be very careful with the terms, otherwise it is nearly impossible to understand what you mean.

```suggestion
  - In case services have overlapping specification an error is yielded.
```